package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "uproxy-lib",
   "description": "Shared libraries for uProxy projects.",
-  "version": "29.0.6",
+  "version": "29.0.7",
```

Comment:
Since this changes the behaviour for existing clients, I think a major version bump is more appropriate.